src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -234,6 +217,7 @@ uct_cuda_copy_mem_alloc_fabric(uct_cuda_copy_md_t *md,
     CUmemAccessDesc access_desc = {};
     ucs_status_t status;
     CUdevice cu_device;
+    CUmemGenericAllocationHandle *generic_handle;
```

Comment:
I don't think even setting ULLONG_MAX works well because typedef may change in the future. Let's leave it as void*/CUmemGenericAllocationHandle* ? After all, this called as part of cuda_copy_mem_alloc_fabric which should be off the critical path.

Reply:
if we don't have a value that can be considered "invalid handle", let's add a flag "uint8_t is_vmm" to the struct